atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraLogHelper.java

```diff
@@ -72,6 +72,13 @@ private static String getUpperEndpoint(Range<LightweightOppToken> range) {
         return range.upperEndpoint().toString();
     }
 
+    public static List<String> tokenRangeHashes(Set<TokenRange> tokenRanges) {
```

Comment:
do we want a way to identify which hash belongs to which host? If I'm not mistaken, we're just going to get a load of hashes but no way to connect which belongs to which host / dc / port

Reply:
That could be useful, but the goals of this PR were primarily to quickly identify whether the rings disagree in terms of the ranges they provide or not (basically if they do then said identification is probably not super helpful because we have severe-data-corruption(TM) at which point we read the unsafe args).

Admittedly this PR drops in value a bit after #5969 as it was written before it was known whether that PR was acceptable, though weird things like "same ranges diff size" which would still be flagged are _probably_ less concerning than "different ranges".